config/environments/test.rb

```diff
@@ -36,9 +36,10 @@
   # Disable request forgery protection in test environment.
   config.action_controller.allow_forgery_protection = false
 
-  # Store uploaded files on the local file system in a temporary directory.
+  # Store uploaded files in the db for testing
```

Comment:
I think we should just delete this comment, especially when it’s already out of sync with storage.yml. :) People can just check storage.yml if they want to know how test is storing files.

I’m definitely in the camp of “comment very little unless it’s really needed to clarify because inevitably the comments will get out of sync from the code.” This is one of those comments that was generated by rails with the original “rails new” and I haven’t gone through the codebase and removed those generated comments which have become irrelevant.

Reply:
yeah agreed with the comments. If the code explains itself without too much trouble then no need for further explanation with comments.

For this file I purposely left it here thinking that we should override anything that would be set in the application.rb config file here since I don't think we want to upload anything to cloud storage during tests. At least that's my initial thought. What do you think?

Reply:
Agreed, we don't want to upload to cloud storage during tests. But since the default is postgres, we shouldn't need to override this for test and instead can use the postgres default in the test environment. I think it's a good thing that tests will use the postgres adapter because if we ever inadvertently break that adapter we'll notice failing tests.